packages/expansion-hub/src/ExpansionHub.ts

```diff
@@ -50,8 +50,21 @@ export interface ExpansionHub extends RevHub {
     // DIO
     setDigitalSingleOutput(dioPin: number, value?: boolean): Promise<void>;
     setDigitalAllOutputs(bitPackedField: number): Promise<void>;
+
+    /**
+     * Set a digital pin as input or output.
+     * @param dioPin
+     * @param direction
+     */
     setDigitalDirection(dioPin: number, direction: DIODirection): Promise<void>;
     getDigitalDirection(dioPin: number): Promise<DIODirection>;
+
+    /**
+     * Read the state of a given digital pin.
+     * Throws an error if the pin is not configured for input
+     *
+     * @param dioPin
+     */
     getDigitalSingleInput(dioPin: number): Promise<boolean>;
```

Comment:
I think I'd like this to return a newly-created `DigitalState` enum:

```typescript
enum DigitalState {
    HIGH,
    LOW
}
```

Controlling a digital output should use the same enum, so let's just add support for digital outputs to this same PR.